index.bs

```diff
@@ -1583,8 +1585,11 @@ Create a named {{MLOperand}} based on a descriptor, that can be used as an input
     1. Let |operand| be the result of invoking the <a>create MLOperand</a> steps with [=this=] and |descriptor|.
         1. If that throws, re-throw the exception and stop.
     1. Set |operand|.{{MLOperand/[[name]]}} to |name|.
-    1. Make a request to the underlying platform to register |operand| as an input and store a reference to the corresponding [=implementation-defined=] platform object in |operand|.{{MLOperand/[[operand]]}}.
-        1. If that fails, throw an "{{OperationError}}" {{DOMException}} and abort these steps.
+    1. If any of the following sub-steps fail, throw an "{{OperationError}}" {{DOMException}} and stop.
+        1. Make a request to the underlying platform to:
+            1. Create an [=implementation-defined=] platform input operand |operandImpl| as an input given |operand|.
```

Comment:
The creation of a platform input operand should depend on operand descriptor for data type, dimensions etc.,

Reply:
That is why I pass the |operand|, and in rest is implementation-defined. I wonder should I just pass the operand.[[descriptor]]? That feels like an implementation detail, and the steps look more complete like this. Can we extract some more generic steps for creating input, or can we leave it in this generic form?